ddl/multi_schema_change.go

```diff
@@ -121,8 +121,11 @@ func onMultiSchemaChange(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job) (ve
 			} else {
 				schemaVersionGenerated = true
 			}
-			ver, err = w.runDDLJob(d, t, &proxyJob)
-			sub.FromProxyJob(&proxyJob, ver)
+			proxyJobVer, err := w.runDDLJob(d, t, &proxyJob)
+			if !proxyJob.MultiSchemaInfo.SkipVersion {
```

Comment:
```suggestion
			if !schemaVersionGenerated {
```

Reply:
The `schemaVersionGenerated` here always equal to true.

Reply:
Improve it by other way, please check again.